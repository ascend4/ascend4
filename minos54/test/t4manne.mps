NAME          MANNE10
ROWS
 G  MON001
 G  MON002
 G  MON003
 G  MON004
 G  MON005
 G  MON006
 G  MON007
 G  MON008
 G  MON009
 G  MON010
 L  CAP002
 L  CAP003
 L  CAP004
 L  CAP005
 L  CAP006
 L  CAP007
 L  CAP008
 L  CAP009
 L  CAP010
 L  TERMINV
COLUMNS
    KAP001    MON001      .1           CAP001     1.0
    KAP001    CAP002    -1.0
    KAP002    MON002      .1           CAP002     1.0
    KAP002    CAP003    -1.0
    KAP003    MON003      .1           CAP003     1.0
    KAP003    CAP004    -1.0
    KAP004    MON004      .1           CAP004     1.0
    KAP004    CAP005    -1.0
    KAP005    MON005      .1           CAP005     1.0
    KAP005    CAP006    -1.0
    KAP006    MON006      .1           CAP006     1.0
    KAP006    CAP007    -1.0
    KAP007    MON007      .1           CAP007     1.0
    KAP007    CAP008    -1.0
    KAP008    MON008      .1           CAP008     1.0
    KAP008    CAP009    -1.0
    KAP009    MON009      .1           CAP009     1.0
    KAP009    CAP010    -1.0
    KAP010    MON010      .1           CAP010     1.0
    KAP010    TERMINV     .03
    CON001    MON001    -1.0
    CON002    MON002    -1.0
    CON003    MON003    -1.0
    CON004    MON004    -1.0
    CON005    MON005    -1.0
    CON006    MON006    -1.0
    CON007    MON007    -1.0
    CON008    MON008    -1.0
    CON009    MON009    -1.0
    CON010    MON010    -1.0
    INV001    MON001    -1.0           CAP002    -1.0
    INV002    MON002    -1.0           CAP003    -1.0
    INV003    MON003    -1.0           CAP004    -1.0
    INV004    MON004    -1.0           CAP005    -1.0
    INV005    MON005    -1.0           CAP006    -1.0
    INV006    MON006    -1.0           CAP007    -1.0
    INV007    MON007    -1.0           CAP008    -1.0
    INV008    MON008    -1.0           CAP009    -1.0
    INV009    MON009    -1.0           CAP010    -1.0
    INV010    MON010    -1.0           CAP011    -1.0
    INV010    TERMINV   -1.0
RHS
*
*   The RHS is zero
*
    LAGRANGE  MON002      -0.9         MON003      -0.8
    LAGRANGE  MON010     -10.0
RANGES
    RANGE1    MON010      10.0         TERMINV     20.0
BOUNDS
 FX BOUND1    KAP001      3.05
 LO BOUND1    KAP002      3.05
 LO BOUND1    KAP003      3.05
 LO BOUND1    KAP004      3.05
 LO BOUND1    KAP005      3.05
 LO BOUND1    KAP006      3.05
 LO BOUND1    KAP007      3.05
 LO BOUND1    KAP008      3.05
 LO BOUND1    KAP009      3.05
 LO BOUND1    KAP010      3.05
 LO BOUND1    CON001       .95
 LO BOUND1    CON002       .95
 LO BOUND1    CON003       .95
 LO BOUND1    CON004       .95
 LO BOUND1    CON005       .95
 LO BOUND1    CON006       .95
 LO BOUND1    CON007       .95
 LO BOUND1    CON008       .95
 LO BOUND1    CON009       .95
 LO BOUND1    CON010       .95
 LO BOUND1    INV001       .05
 LO BOUND1    INV002       .05
 LO BOUND1    INV003       .05
 LO BOUND1    INV004       .05
 LO BOUND1    INV005       .05
 LO BOUND1    INV006       .05
 LO BOUND1    INV007       .05
 LO BOUND1    INV008       .05
 LO BOUND1    INV009       .05
 LO BOUND1    INV010       .05
 UP BOUND1    INV008       .112
 UP BOUND1    INV009       .114
 UP BOUND1    INV010       .116
 FX INITIAL   KAP002      3.1
 FX INITIAL   KAP003      3.2
 FX INITIAL   KAP004      3.3
 FX INITIAL   KAP005      3.4
 FX INITIAL   KAP006      3.5
 FX INITIAL   KAP007      3.6
 FX INITIAL   KAP008      3.7
 FX INITIAL   KAP009      3.8
 FX INITIAL   KAP010      3.9
ENDATA
