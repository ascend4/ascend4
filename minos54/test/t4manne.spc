Begin t4manne  (10-period economic growth model)

   Problem number          1114
   Maximize
   Rows                     100
   Columns                  100
   Elements                 100
   Upper bound            100.0
   Objective =           funobj

   Nonlinear constraints     10
   Nonlinear Jacobian  vars  10
   Nonlinear objective vars  20

   MPS file                  10
*  New Basis file            11

   Jacobian              Sparse
   Major iterations           8
   Minor iterations          20
   Penalty parameter        0.1

   Hessian dimension         10
*  Derivative level           0
*  Verify gradients

   Scale option               2
   Iterations                50
   Print level (jflxb)    00000
   Print frequency            1
   Summary level              0
   Summary frequency          1
End Manne10
