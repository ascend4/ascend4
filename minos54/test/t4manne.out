1
 
          
         ==============================
         M I N O S    5.4    (Dec 1992)
         ==============================

      Begin t4manne  (10-period economic growth model)
       
         Problem number          1114
         Maximize
         Rows                     100
         Columns                  100
         Elements                 100
         Upper bound            100.0
         Objective =           funobj
       
         Nonlinear constraints     10
         Nonlinear Jacobian  vars  10
         Nonlinear objective vars  20
       
         MPS file                  10
      *  New Basis file            11
       
         Jacobian              Sparse
         Major iterations           8
         Minor iterations          20
         Penalty parameter        0.1
       
         Hessian dimension         10
      *  Derivative level           0
      *  Verify gradients
       
         Scale option               2
         Iterations                50
         Print level (jflxb)    00000
         Print frequency            1
         Summary level              0
         Summary frequency          1
      End Manne10

 Nonzeros allowed for in LU factors    24650
 Scale option  2,      Partial price       1
 Partial price section size (A)           30
 Partial price section size (I)           20



 Matrix Statistics
 -----------------
               Total      Normal        Free       Fixed     Bounded
 Rows             20          18           0           0           2
 Columns          30           0           0           1          29

 No. of matrix elements                   59     Density       9.833
 Biggest                          1.0000E+00  (excluding fixed columns,
 Smallest                         3.0000E-02   free rows, and RHS)

 No. of objective coefficients             0


 Number of Nonlinear constraints          10
 Number of Linear    constraints          10
 Number of Variables                      30
1
 
 Initial basis
 -------------

 No basis file supplied


 This is problem  t4manne.   nt =  10   b =    .250



 funcon  sets      10   out of      10   constraint gradients.

 Cheap test on funcon...
 The Jacobian seems to be OK.
 The largest discrepancy was    6.59E-10  in constraint     2



 funobj  sets      20   out of      20   objective  gradients.

 Cheap test on funobj...
 The objective gradients seem to be OK.
 Gradient projected in two directions   4.00258220426E+00   1.00000000000E+00
 Difference approximations              4.00257400295E+00   9.99999843744E-01


 Scaling
 -------
             Min elem    Max elem       Max col ratio
 After  0    3.00E-02    1.00E+00               33.33
 After  1    4.16E-01    2.40E+00                5.77
 After  2    5.06E-01    1.98E+00                3.90
 After  3    5.17E-01    1.93E+00                3.73

            Min scale                       Max scale      Between 0.5 and 2.0
 Col     10   5.0E-01            Col     30   6.7E+00               19    63.3
 Row     20   1.1E-01            Row     11   1.5E+00               10    50.0

 Norm of fixed columns and slacks             1.8E+00
 (before and after row scaling)               5.7E+00


 Crash option  3
 Crash on linear E  rows:
1
 
 Iterations
 ----------

 Major minor   total ninf  true objective    viol    rg    nsb   ncon xchange lchange    step     LU penalty
     0     0       0    0  0.00000000E+00 0.0E+00 0.0E+00    9      3 0.0E+00 0.0E+00 1.0E+00      0 1.0E-01
 Crash on linear LG rows:
 Slacks     0  Free cols     0  Preferred     0
 Unit      10  Double        0  Triangle      0  Pad     0
     1     1T      1    0  0.00000000E+00 0.0E+00 0.0E+00    8      4 0.0E+00 0.0E+00 1.0E+00     31 1.0E-01
 Crash on nonlinear rows:
 Slacks     0  Free cols     0  Preferred     0
 Unit      10  Double        0  Triangle      0  Pad     0
     2     6       7    0  2.67005654E+00 6.3E-06 1.2E-01    7     16 3.4E-03 2.7E+01 1.0E+00     33 1.0E-01
     3     2       9    0  2.67010154E+00 5.4E-07 2.1E-03    7     21 1.2E-03 3.2E-03 1.0E+00     33 1.0E-01
     4     2      11    0  2.67009863E+00 1.7E-09 1.2E-06    7     24 4.3E-05 7.6E-05 1.0E+00     33 1.0E-02
1
 
 EXIT -- optimal solution found

 manne10 
 No. of iterations                  11   Objective value      2.6700986315E+00
 No. of major iterations             4   Linear objective     0.0000000000E+00
 Penalty parameter             .001000   Nonlinear objective  2.6700986315E+00
 No. of calls to funobj             24   No. of calls to funcon             24
 No. of superbasics                  7   Norm of reduced gradient    1.248E-06
 No. of basic nonlinears            11   Norm rg / Norm pi           7.456E-08
 No. of degenerate steps             0   Percentage                        .00
 Norm of x  (scaled)         1.454E+00   Norm of pi (scaled)         1.674E+01
 Norm of x                   6.534E+00   Norm of pi                  7.609E+00
 Primal inf (scaled)      0    0.0E+00   Dual inf   (scaled)      9    1.2E-06
 Primal infeas            0    0.0E+00   Dual infeas              9    1.2E-07
 Constraint violation        2.750E-09   Normalized                  3.650E-10
1
 
 NAME           manne10               OBJECTIVE VALUE       2.6700986315E+00

 STATUS         OPTIMAL SOLN          ITERATION     11    SUPERBASICS      7

 OBJECTIVE      funobj   (Max)
 RHS            zero    
 RANGES         range1  
 BOUNDS         bound1  

 SECTION 1 - ROWS

  NUMBER  ...ROW.. STATE  ...ACTIVITY...  SLACK ACTIVITY  ..LOWER LIMIT.  ..UPPER LIMIT.  .DUAL ACTIVITY    ..I

      31  r      1    LL          .00000          .00000          .00000           NONE         -1.01064      1
      32  r      2    LL          .00000          .00000          .00000           NONE          -.93193      2
      33  r      3    LL          .00000          .00000          .00000           NONE          -.85926      3
      34  r      4    LL          .00000          .00000          .00000           NONE          -.79217      4
      35  r      5    LL          .00000          .00000          .00000           NONE          -.73021      5
      36  r      6    LL          .00000          .00000          .00000           NONE          -.67299      6
      37  r      7    LL          .00000          .00000          .00000           NONE          -.62015      7
      38  r      8    LL          .00000          .00000          .00000           NONE          -.57134      8
      39  r      9    LL          .00000          .00000          .00000           NONE          -.52625      9
      40  r     10    LL          .00000          .00000          .00000        10.00000        -9.86433     10
      41  r     11    UL          .00000          .00000           NONE           .00000         1.01064     11
      42  r     12    UL          .00000          .00000           NONE           .00000          .93193     12
      43  r     13    UL          .00000          .00000           NONE           .00000          .85926     13
      44  r     14    UL          .00000          .00000           NONE           .00000          .79217     14
      45  r     15    UL          .00000          .00000           NONE           .00000          .73021     15
      46  r     16    UL          .00000          .00000           NONE           .00000          .67299     16
      47  r     17    UL          .00000          .00000           NONE           .00000          .62015     17
      48  r     18    UL          .00000          .00000           NONE           .00000          .57134     18
      49  r     19    UL          .00000          .00000           NONE           .00000          .52625     19
      50  r     20    UL          .00000          .00000       -20.00000          .00000        10.73212     20
1
 
 SECTION 2 - COLUMNS

  NUMBER  .COLUMN. STATE  ...ACTIVITY...  .OBJ GRADIENT.  ..LOWER LIMIT.  ..UPPER LIMIT.  REDUCED GRADNT    M+J

       1  x      1    EQ         3.05000          .00000         3.05000         3.05000         1.09568     21
       2  x      2    BS         3.12665          .00000         3.05000           NONE           .00000     22
       3  x      3   SBS         3.21443          .00000         3.05000           NONE           .00000     23
       4  x      4   SBS         3.30400          .00000         3.05000           NONE           .00000     24
       5  x      5   SBS         3.39522          .00000         3.05000           NONE           .00000     25
       6  x      6   SBS         3.48788          .00000         3.05000           NONE           .00000     26
       7  x      7   SBS         3.58172          .00000         3.05000           NONE           .00000     27
       8  x      8   SBS         3.67643          .00000         3.05000           NONE           .00000     28
       9  x      9   SBS         3.77158          .00000         3.05000           NONE           .00000     29
      10  x     10    BS         3.86667          .00000         3.05000           NONE           .00000     30
      11  x     11    LL          .95000         1.00000          .95000           NONE          -.01064     31
      12  x     12    BS          .96842          .93193          .95000           NONE           .00000     32
      13  x     13    BS          .99780          .85926          .95000           NONE           .00000     33
      14  x     14    BS         1.02820          .79217          .95000           NONE           .00000     34
      15  x     15    BS         1.05967          .73021          .95000           NONE           .00000     35
      16  x     16    BS         1.09227          .67299          .95000           NONE           .00000     36
      17  x     17    BS         1.12608          .62015          .95000           NONE           .00000     37
      18  x     18    BS         1.16116          .57134          .95000           NONE           .00000     38
      19  x     19    BS         1.19763          .52625          .95000           NONE           .00000     39
      20  x     20    BS         1.21394         9.86433          .95000           NONE           .00000     40
      21  x     21    BS          .07665          .00000          .05000           NONE           .00000     41
      22  x     22    BS          .08778          .00000          .05000           NONE           .00000     42
      23  x     23    BS          .08957          .00000          .05000           NONE           .00000     43
      24  x     24    BS          .09122          .00000          .05000           NONE           .00000     44
      25  x     25    BS          .09266          .00000          .05000           NONE           .00000     45
      26  x     26    BS          .09384          .00000          .05000           NONE           .00000     46
      27  x     27    BS          .09471          .00000          .05000           NONE           .00000     47
      28  x     28    BS          .09515          .00000          .05000          .11200          .00000     48
      29  x     29    BS          .09508          .00000          .05000          .11400          .00000     49
      30  x     30    UL          .11600          .00000          .05000          .11600          .86778     50

 funcon called with nstate =   2


 Final nonlinear function values
     1.02665     1.05620     1.08738     1.11942     1.15233
     1.18612     1.22078     1.25632     1.29271     1.32994

 funobj called with nstate =   2
       
      Derivative level 3
      Print level 0
      Verify level 0
      Scale option 0
      Iterations                                                            20
      Penalty parameter                                         1.00000000E-02

 Nonzeros allowed for in LU factors    24685
 Scale option  0,      Partial price       1
 Partial price section size (A)           30
 Partial price section size (I)           20



 Matrix Statistics
 -----------------
               Total      Normal        Free       Fixed     Bounded
 Rows             20          18           0           0           2
 Columns          30           0           0           1          29

 No. of matrix elements                   59     Density       9.833
 Biggest                          1.0000E+00  (excluding fixed columns,
 Smallest                         3.0000E-02   free rows, and RHS)

 No. of objective coefficients             0


 Number of Nonlinear constraints          10
 Number of Linear    constraints          10
 Number of Variables                      30
1
 
 Initial basis
 -------------


 This is problem  t4manne.   nt =  10   b =    .250



 funcon  sets      10   out of      10   constraint gradients.

 Cheap test on funcon...
 The Jacobian seems to be OK.
 The largest discrepancy was    6.63E-10  in constraint     3



 funobj  sets      20   out of      20   objective  gradients.

 Cheap test on funobj...
 The objective gradients seem to be OK.
 Gradient projected in two directions  -1.02839772604E+00   8.28431937678E-01
 Difference approximations             -1.02840429024E+00   8.28431826599E-01
1
 
 Iterations
 ----------

 Major minor   total ninf  true objective    viol    rg    nsb   ncon xchange lchange    step     LU penalty
     0     0       0    0  0.00000000E+00 2.7E-09 0.0E+00    7      3 0.0E+00 0.0E+00 1.0E+00      0 1.0E-02
     1     0T      0    0  0.00000000E+00 1.1E-16 0.0E+00    7      4 0.0E+00 0.0E+00 1.0E+00     33 1.0E-02
     2     0       0    0  2.67009863E+00 1.1E-16 1.2E-07    7      5 0.0E+00 2.1E-10 1.0E+00     33 1.0E-02
     3     0       0    0  2.67009863E+00 1.1E-16 1.2E-07    7      6 0.0E+00 0.0E+00 1.0E+00     33 1.0E-03
     4     0       0    0  2.67009863E+00 1.1E-16 1.2E-07    7      7 0.0E+00 0.0E+00 1.0E+00     33 1.0E-04
1
 
 EXIT -- optimal solution found

 manne10 
 No. of iterations                   0   Objective value      2.6700986272E+00
 No. of major iterations             4   Linear objective     0.0000000000E+00
 Penalty parameter             .000010   Nonlinear objective  2.6700986272E+00
 No. of calls to funobj              7   No. of calls to funcon              7
 No. of superbasics                  7   Norm of reduced gradient    1.165E-07
 No. of basic nonlinears            11   Norm rg / Norm pi           1.531E-08
 No. of degenerate steps             0   Percentage                        .00
 Norm of x                   6.534E+00   Norm of pi                  7.609E+00
 Primal infeas            0    0.0E+00   Dual infeas              9    1.2E-07
 Constraint violation        1.110E-16   Normalized                  1.474E-17
1
 
 NAME           manne10               OBJECTIVE VALUE       2.6700986272E+00

 STATUS         OPTIMAL SOLN          ITERATION      0    SUPERBASICS      7

 OBJECTIVE      funobj   (Max)
 RHS            zero    
 RANGES         range1  
 BOUNDS         bound1  

 SECTION 1 - ROWS

  NUMBER  ...ROW.. STATE  ...ACTIVITY...  SLACK ACTIVITY  ..LOWER LIMIT.  ..UPPER LIMIT.  .DUAL ACTIVITY    ..I

      31  r      1    LL          .00000          .00000          .00000           NONE         -1.01064      1
      32  r      2    LL          .00000          .00000          .00000           NONE          -.93193      2
      33  r      3    LL          .00000          .00000          .00000           NONE          -.85926      3
      34  r      4    LL          .00000          .00000          .00000           NONE          -.79217      4
      35  r      5    LL          .00000          .00000          .00000           NONE          -.73021      5
      36  r      6    LL          .00000          .00000          .00000           NONE          -.67299      6
      37  r      7    LL          .00000          .00000          .00000           NONE          -.62015      7
      38  r      8    LL          .00000          .00000          .00000           NONE          -.57134      8
      39  r      9    LL          .00000          .00000          .00000           NONE          -.52625      9
      40  r     10    LL          .00000          .00000          .00000        10.00000        -9.86433     10
      41  r     11    UL          .00000          .00000           NONE           .00000         1.01064     11
      42  r     12    UL          .00000          .00000           NONE           .00000          .93193     12
      43  r     13    UL          .00000          .00000           NONE           .00000          .85926     13
      44  r     14    UL          .00000          .00000           NONE           .00000          .79217     14
      45  r     15    UL          .00000          .00000           NONE           .00000          .73021     15
      46  r     16    UL          .00000          .00000           NONE           .00000          .67299     16
      47  r     17    UL          .00000          .00000           NONE           .00000          .62015     17
      48  r     18    UL          .00000          .00000           NONE           .00000          .57134     18
      49  r     19    UL          .00000          .00000           NONE           .00000          .52625     19
      50  r     20    UL          .00000          .00000       -20.00000          .00000        10.73212     20
1
 
 SECTION 2 - COLUMNS

  NUMBER  .COLUMN. STATE  ...ACTIVITY...  .OBJ GRADIENT.  ..LOWER LIMIT.  ..UPPER LIMIT.  REDUCED GRADNT    M+J

       1  x      1    EQ         3.05000          .00000         3.05000         3.05000         1.09568     21
       2  x      2    BS         3.12665          .00000         3.05000           NONE           .00000     22
       3  x      3   SBS         3.21443          .00000         3.05000           NONE           .00000     23
       4  x      4   SBS         3.30400          .00000         3.05000           NONE           .00000     24
       5  x      5   SBS         3.39522          .00000         3.05000           NONE           .00000     25
       6  x      6   SBS         3.48788          .00000         3.05000           NONE           .00000     26
       7  x      7   SBS         3.58172          .00000         3.05000           NONE           .00000     27
       8  x      8   SBS         3.67643          .00000         3.05000           NONE           .00000     28
       9  x      9   SBS         3.77158          .00000         3.05000           NONE           .00000     29
      10  x     10    BS         3.86667          .00000         3.05000           NONE           .00000     30
      11  x     11    LL          .95000         1.00000          .95000           NONE          -.01064     31
      12  x     12    BS          .96842          .93193          .95000           NONE           .00000     32
      13  x     13    BS          .99780          .85926          .95000           NONE           .00000     33
      14  x     14    BS         1.02820          .79217          .95000           NONE           .00000     34
      15  x     15    BS         1.05967          .73021          .95000           NONE           .00000     35
      16  x     16    BS         1.09227          .67299          .95000           NONE           .00000     36
      17  x     17    BS         1.12608          .62015          .95000           NONE           .00000     37
      18  x     18    BS         1.16116          .57134          .95000           NONE           .00000     38
      19  x     19    BS         1.19763          .52625          .95000           NONE           .00000     39
      20  x     20    BS         1.21394         9.86433          .95000           NONE           .00000     40
      21  x     21    BS          .07665          .00000          .05000           NONE           .00000     41
      22  x     22    BS          .08778          .00000          .05000           NONE           .00000     42
      23  x     23    BS          .08957          .00000          .05000           NONE           .00000     43
      24  x     24    BS          .09122          .00000          .05000           NONE           .00000     44
      25  x     25    BS          .09266          .00000          .05000           NONE           .00000     45
      26  x     26    BS          .09384          .00000          .05000           NONE           .00000     46
      27  x     27    BS          .09471          .00000          .05000           NONE           .00000     47
      28  x     28    BS          .09515          .00000          .05000          .11200          .00000     48
      29  x     29    BS          .09508          .00000          .05000          .11400          .00000     49
      30  x     30    UL          .11600          .00000          .05000          .11600          .86778     50

 funcon called with nstate =   2


 Final nonlinear function values
     1.02665     1.05620     1.08738     1.11942     1.15233
     1.18612     1.22078     1.25632     1.29271     1.32994

 funobj called with nstate =   2

 Nonzeros allowed for in LU factors    24685

 gotbas = T     gotfac = F     gothes = F     gotscl = F
 Scale option  0,      Partial price       1
 Partial price section size (A)           30
 Partial price section size (I)           20



 Matrix Statistics
 -----------------
               Total      Normal        Free       Fixed     Bounded
 Rows             20          18           0           0           2
 Columns          30           0           0           1          29

 No. of matrix elements                   59     Density       9.833
 Biggest                          1.0000E+00  (excluding fixed columns,
 Smallest                         3.0000E-02   free rows, and RHS)

 No. of objective coefficients             0


 Number of Nonlinear constraints          10
 Number of Linear    constraints          10
 Number of Variables                      30
1
 
 Initial basis
 -------------


 This is problem  t4manne.   nt =  10   b =    .250



 funcon  sets      10   out of      10   constraint gradients.

 Cheap test on funcon...
 The Jacobian seems to be OK.
 The largest discrepancy was    6.63E-10  in constraint     3



 funobj  sets      20   out of      20   objective  gradients.

 Cheap test on funobj...
 The objective gradients seem to be OK.
 Gradient projected in two directions  -1.02839772770E+00   8.28431937866E-01
 Difference approximations             -1.02840429178E+00   8.28431826938E-01
1
 
 Iterations
 ----------

 Major minor   total ninf  true objective    viol    rg    nsb   ncon xchange lchange    step     LU penalty
     0     0       0    0  0.00000000E+00 1.1E-16 0.0E+00    7      3 0.0E+00 0.0E+00 1.0E+00      0 1.0E-05
     1     0T      0    0  0.00000000E+00 1.1E-16 0.0E+00    7      4 0.0E+00 0.0E+00 1.0E+00     33 1.0E-05
     2     0       0    0  2.67009863E+00 1.1E-16 1.2E-07    7      5 0.0E+00 0.0E+00 1.0E+00     33 1.0E-05
     3     0       0    0  2.67009863E+00 1.1E-16 1.2E-07    7      6 0.0E+00 0.0E+00 1.0E+00     33 1.0E-06
     4     0       0    0  2.67009863E+00 1.1E-16 1.2E-07    7      7 0.0E+00 0.0E+00 1.0E+00     33 0.0E+00
1
 
 EXIT -- optimal solution found

 manne10 
 No. of iterations                   0   Objective value      2.6700986272E+00
 No. of major iterations             4   Linear objective     0.0000000000E+00
 Penalty parameter             .000000   Nonlinear objective  2.6700986272E+00
 No. of calls to funobj              7   No. of calls to funcon              7
 No. of superbasics                  7   Norm of reduced gradient    1.165E-07
 No. of basic nonlinears            11   Norm rg / Norm pi           1.531E-08
 No. of degenerate steps             0   Percentage                        .00
 Norm of x                   6.534E+00   Norm of pi                  7.609E+00
 Primal infeas            0    0.0E+00   Dual infeas              9    1.2E-07
 Constraint violation        1.110E-16   Normalized                  1.474E-17
1
 
 NAME           manne10               OBJECTIVE VALUE       2.6700986272E+00

 STATUS         OPTIMAL SOLN          ITERATION      0    SUPERBASICS      7

 OBJECTIVE      funobj   (Max)
 RHS            zero    
 RANGES         range1  
 BOUNDS         bound1  

 SECTION 1 - ROWS

  NUMBER  ...ROW.. STATE  ...ACTIVITY...  SLACK ACTIVITY  ..LOWER LIMIT.  ..UPPER LIMIT.  .DUAL ACTIVITY    ..I

      31  r      1    LL          .00000          .00000          .00000           NONE         -1.01064      1
      32  r      2    LL          .00000          .00000          .00000           NONE          -.93193      2
      33  r      3    LL          .00000          .00000          .00000           NONE          -.85926      3
      34  r      4    LL          .00000          .00000          .00000           NONE          -.79217      4
      35  r      5    LL          .00000          .00000          .00000           NONE          -.73021      5
      36  r      6    LL          .00000          .00000          .00000           NONE          -.67299      6
      37  r      7    LL          .00000          .00000          .00000           NONE          -.62015      7
      38  r      8    LL          .00000          .00000          .00000           NONE          -.57134      8
      39  r      9    LL          .00000          .00000          .00000           NONE          -.52625      9
      40  r     10    LL          .00000          .00000          .00000        10.00000        -9.86433     10
      41  r     11    UL          .00000          .00000           NONE           .00000         1.01064     11
      42  r     12    UL          .00000          .00000           NONE           .00000          .93193     12
      43  r     13    UL          .00000          .00000           NONE           .00000          .85926     13
      44  r     14    UL          .00000          .00000           NONE           .00000          .79217     14
      45  r     15    UL          .00000          .00000           NONE           .00000          .73021     15
      46  r     16    UL          .00000          .00000           NONE           .00000          .67299     16
      47  r     17    UL          .00000          .00000           NONE           .00000          .62015     17
      48  r     18    UL          .00000          .00000           NONE           .00000          .57134     18
      49  r     19    UL          .00000          .00000           NONE           .00000          .52625     19
      50  r     20    UL          .00000          .00000       -20.00000          .00000        10.73212     20
1
 
 SECTION 2 - COLUMNS

  NUMBER  .COLUMN. STATE  ...ACTIVITY...  .OBJ GRADIENT.  ..LOWER LIMIT.  ..UPPER LIMIT.  REDUCED GRADNT    M+J

       1  x      1    EQ         3.05000          .00000         3.05000         3.05000         1.09568     21
       2  x      2    BS         3.12665          .00000         3.05000           NONE           .00000     22
       3  x      3   SBS         3.21443          .00000         3.05000           NONE           .00000     23
       4  x      4   SBS         3.30400          .00000         3.05000           NONE           .00000     24
       5  x      5   SBS         3.39522          .00000         3.05000           NONE           .00000     25
       6  x      6   SBS         3.48788          .00000         3.05000           NONE           .00000     26
       7  x      7   SBS         3.58172          .00000         3.05000           NONE           .00000     27
       8  x      8   SBS         3.67643          .00000         3.05000           NONE           .00000     28
       9  x      9   SBS         3.77158          .00000         3.05000           NONE           .00000     29
      10  x     10    BS         3.86667          .00000         3.05000           NONE           .00000     30
      11  x     11    LL          .95000         1.00000          .95000           NONE          -.01064     31
      12  x     12    BS          .96842          .93193          .95000           NONE           .00000     32
      13  x     13    BS          .99780          .85926          .95000           NONE           .00000     33
      14  x     14    BS         1.02820          .79217          .95000           NONE           .00000     34
      15  x     15    BS         1.05967          .73021          .95000           NONE           .00000     35
      16  x     16    BS         1.09227          .67299          .95000           NONE           .00000     36
      17  x     17    BS         1.12608          .62015          .95000           NONE           .00000     37
      18  x     18    BS         1.16116          .57134          .95000           NONE           .00000     38
      19  x     19    BS         1.19763          .52625          .95000           NONE           .00000     39
      20  x     20    BS         1.21394         9.86433          .95000           NONE           .00000     40
      21  x     21    BS          .07665          .00000          .05000           NONE           .00000     41
      22  x     22    BS          .08778          .00000          .05000           NONE           .00000     42
      23  x     23    BS          .08957          .00000          .05000           NONE           .00000     43
      24  x     24    BS          .09122          .00000          .05000           NONE           .00000     44
      25  x     25    BS          .09266          .00000          .05000           NONE           .00000     45
      26  x     26    BS          .09384          .00000          .05000           NONE           .00000     46
      27  x     27    BS          .09471          .00000          .05000           NONE           .00000     47
      28  x     28    BS          .09515          .00000          .05000          .11200          .00000     48
      29  x     29    BS          .09508          .00000          .05000          .11400          .00000     49
      30  x     30    UL          .11600          .00000          .05000          .11600          .86778     50

 funcon called with nstate =   2


 Final nonlinear function values
     1.02665     1.05620     1.08738     1.11942     1.15233
     1.18612     1.22078     1.25632     1.29271     1.32994

 funobj called with nstate =   2
